packages/contract-schema/test/solc.js

```diff
@@ -88,9 +88,8 @@ describe("solc", function () {
       );
     });
 
-    //check that ast and legacyAST have the correct form
+    //check that ast has the correct form
```

Comment:
The check for the legacyAST is removed from this file because the legacy AST has been removed from the **solc**  **0.8.0** onwards. Can you please check if this can be done, @haltman-at ?

Reply:
Not sure what you're asking.  I agree it makes sense to remove that check though.

Reply:
Oh, sorry if I am not clear about what I asked. I just wanted your confirmation if it is okay to remove the legacy AST check. Thanks for explaining that @haltman-at .